go.mod

```diff
@@ -249,9 +250,64 @@ require (
 replace (
 	bazil.org/fuse => bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898
 	bitbucket.org/bertimus9/systemstat => bitbucket.org/bertimus9/systemstat v0.5.0
-	cloud.google.com/go => cloud.google.com/go v0.97.0
-	cloud.google.com/go/bigquery => cloud.google.com/go/bigquery v1.8.0
-	cloud.google.com/go/storage => cloud.google.com/go/storage v1.10.0
+	cloud.google.com/go => cloud.google.com/go v0.104.0
```

Comment:
this looks like it's going to be a painful nexus going forward... :-/

Reply:
opened https://github.com/kubernetes/kubernetes/issues/113366 to see if we can resolve this